ddl/serial_test.go

```diff
@@ -240,101 +240,6 @@ func (s *testSerialSuite) TestRecoverTableByJobID(c *C) {
 	c.Assert(gcEnable, Equals, false)
 }
 
-func (s *testSerialSuite) TestRecoverTableByTableName(c *C) {
```

Comment:
This UT is moved to package `executor` because:
1. Code about this test is almost in `executor/ddl.go` and 
2. UT coverage of package `ddl` doesn't decrease.